packages/site-docs/src/styles/_sections.scss

```diff
@@ -245,6 +245,13 @@
   display: inline-block;
 }
 
+#{example("NumericInputBasicExample")} {
+  .docs-react-numeric-input-example {
```

Comment:
please use one or the other: `example()` mixin or `.docs-react-**-example` class. either one should be enough to target your element.